tools/project_manager/edit_common.inc

```diff
@@ -50,7 +55,7 @@ function language_list($language)
 
     $array_list = $GLOBALS['lang_list'];
 
-    echo "<select name='pri_language'>";
+    echo "<select name='pri_language' required>";
     echo "<option value=''>" . _("Primary Language") . "</option>";
```

Comment:
Looking at this afresh it feels weird for the two language fields to specify Primary/Secondary in the drop-down. I feel like this would be better done like:
```
Primary: [ Select One  ]   Secondary: [            ]
```
then we could put `Select One` in the first one and leave the second one blank like we do the other optional fields.

And yes, I realize we're well into feature creep here, so feel free to punt as this could be done later.